cmd/skupper/skupper_podman_service.go

```diff
@@ -423,3 +423,36 @@ func (s *SkupperPodmanService) UnexposeFlags(cmd *cobra.Command) error {
 	cmd.Use = "unexpose [host hostOrIP]"
 	return nil
 }
+
+func (s *SkupperPodmanService) UnExposeArgs(cmd *cobra.Command, args []string) error {
+	address, err := cmd.Flags().GetString("address")
+	if err != nil || address == "" {
+		return fmt.Errorf("--address is required")
+	}
+	if exposeOpts.Address != "" && len(exposeOpts.Ports) == 0 {
+		return fmt.Errorf("--port is required")
+	}
+	if len(args) < 2 {
+		return fmt.Errorf("Target type and target value must all be specified (e.g. 'skupper unexpose <target-type> <target-value>')")
```

Comment:
Maybe create an unexported method used by both `ExposeArgs` and `UnexposeArgs` that
simply gets the command name to compose the error message using `cmd.Name()`.

Reply:
Updated in e9cd7496aa1548fcb4cbb73325afa2ab6347badd